main.go

```diff
@@ -0,0 +1,32 @@
+package main
+
+import (
+	"context"
+	"log"
+
+	"github.com/aws/aws-sdk-go/aws"
+	"github.com/aws/aws-sdk-go/aws/session"
+	"github.com/aws/aws-sdk-go/service/ec2"
+	ec2Waste "github.com/timmyers/cloudwaste/pkg/aws/ec2"
+)
+
+func main() {
+	sess := session.Must(session.NewSession(aws.NewConfig().WithRegion("us-east-1")))
```

Comment:
```suggestion
	sess := session.Must(session.NewSession())
```
Every time I've configured the aws Session instead of configuring the individual service clients, I've come across some insane bug that takes days to track. We should configure the clients instead of the overall session. I've had bugs where serialization fails, I've had bugs where networking stops entirely. It's like, basically unpredictable.